.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: '00:00'
+    timezone: Asia/Calcutta
```

Comment:
The default reviewer is @bencodezen, Ben would you like to choose a proper time and timezone?

Reply:
Yeah that would be helpful.